### PR TITLE
 docs(toggle-switch): Add Toggle Switch CSS docs

### PR DESCRIPTION
**Pull request by matuzalemsteles**

### CODE REVIEW

File: next.clayui.com/content/docs/css/components/forms/text-input-group.md [hunk 1]
```diff
@@ -0,0 +1,654 @@
...
+		<div class="input-group-append input-group-item input-group-item-shrink">
+			<button class="btn btn-secondary" type="submit">Submit</button>
+		</div>
+	</div>
+</div>
+```
+
+## Inset
+
+You can insert buttons and links into input group items with the following helper classes: .input-group-inset, `.input-group-inset-before`, `.input-group-inset-after`, `.input-group-inset-item`, `.input-group-inset-item-before`, and `.input-group-inset-item-after.` Use the -before classes to inset the button/link at the beginning of the input group, or use the -after classes to inset the button/link at the end of the input group.
```
Comment (bryceosterhaus) [inline next.clayui.com/content/docs/css/components/forms/text-input-group.md]: I think ".input-group-inset" needs backticks around it `.input-group-inset`

File: next.clayui.com/content/docs/css/components/forms/text-input-group.md [hunk 2]
```diff
@@ -0,0 +1,654 @@
...
+
+<div class="clay-site-alert alert alert-warning">
+	To retain the focus styles around the `.input-group-inset` component when a button or link inside is focused, add the focus class to the `.input-group-item`. Focus Box Shadows are only supported in `.input-group-inset` with `.input-group-item.focus`. `.input-prepend` and .input-append aren't supported with `.input-group-inset`.
```
Comment (bryceosterhaus) [inline next.clayui.com/content/docs/css/components/forms/text-input-group.md]: 'input-append' needs backticks

File: next.clayui.com/content/docs/css/components/forms/text-input-multi-select.md [hunk 3]
```diff
@@ -0,0 +1,515 @@
+---
+title: "Multi Select"
+description: "Multi select is the field type that allows writing text to create “tags” that are represented in the shape of labels. This components counts with auto complete feature."
```
Comment (bryceosterhaus) [inline next.clayui.com/content/docs/css/components/forms/text-input-multi-select.md]: This phrasing is a little confusing "This components counts with auto complete feature."

File: next.clayui.com/content/docs/css/components/forms/text-input.md [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+				<use href="/images/icons/icons.svg#asterisk" />
+			</svg>
+		</label>
+		<input class="form-control" id="inputWarning1" type="text"/>
+		<div class="form-feedback-group">
+			<div class="form-feedback-item">This is a form-feedback-item.</div>
+			<div class="form-feedback-item">
+				<span class="form-feedback-indicator">
+					<svg class="lexicon-icon lexicon-icon-warning-full" focusable="false" role="presentation">
+						`<use href="/images/icons/icons.svg#warning-full" />
```
Comment (bryceosterhaus) [inline next.clayui.com/content/docs/css/components/forms/text-input.md]: should this backtick be here?

File: next.clayui.com/content/docs/css/components/forms/text-input.md [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+			<use href="/images/icons/icons.svg#asterisk" />
+		</svg>
+	</label>
+	<input class="form-control" id="inputWarning1" type="text"/>
+	<div class="form-feedback-group">
+		<div class="form-feedback-item">This is a form-feedback-item.</div>
+		<div class="form-feedback-item">
+			<span class="form-feedback-indicator">
+				<svg class="lexicon-icon lexicon-icon-warning-full" focusable="false" role="presentation">
+					`<use href="/images/icons/icons.svg#warning-full" />
```
Comment (bryceosterhaus) [inline next.clayui.com/content/docs/css/components/forms/text-input.md]: ^ same as above

**Comment by bryceosterhaus** [comment]
btw, great job on the preview routes in netlify!

File: next.clayui.com/content/docs/css/components/forms/text-input-multi-select.md (hunk 3 above)
Comment (matuzalemsteles) [reply next.clayui.com/content/docs/css/components/forms/text-input-multi-select.md]: Hm yeah, I'm going to remove that.

File: next.clayui.com/content/docs/css/components/forms/text-input.md (hunk 4 above)
Comment (matuzalemsteles) [reply next.clayui.com/content/docs/css/components/forms/text-input.md]: Oh no, I ended up putting it on without meaning to. Thanks for catch

**Comment by matuzalemsteles** [comment]
Thanks @bryceosterhaus, update.

**Pull request merged**